README.md

```diff
@@ -218,6 +218,7 @@ The image has the following extensions pre-installed, there extensions can be en
 * OpenGraphMeta
 * OpenIDConnect
 * PageExchange
+* PageForms
```

Comment:
Almost the same as SMW, the extension is bundled, but it can't be enabled via environment variable, so probably worth a special mention and move to a separate (new) list below

Reply:
Yep, that makes sense!